astroid/builder.py

```diff
@@ -157,6 +160,10 @@ def _post_build(self, module, encoding):
 
         # Visit the transforms
         if self._apply_transforms:
+            if modutils.is_module_name_part_of_extension_package_whitelist(
+                module.name, self._manager.extension_package_whitelist
+            ):
+                return module
```

Comment:
```suggestion
            if module.is_in_extension_package_whitelist(
                self._manager.extension_package_whitelist
            ):
                return module
```
If we create a function in the `nodes.Module` class it permits to have auto-completion later when we'll have typing and it put the function where it's expected to be.

Reply:
I'm not sure this function should be part of `nodes.Module`. I see it more as a helper function for which `modutils` is actually a pretty good place already.

Reply:
If a function only works for a specific class, why create  a standalone function that will evade autocomplete on instance of the class and be hard to find ?

Reply:
It doesn't depend on an instance of `nodes.Module`. The first argument is just the module name as string.

I would think of it differently: In case you have a `Module` node in pylint, are you likely to need to call this function? I think no, it's just a helper for astroid.

Reply:
OK this make sense.